src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/__test__/DiaryOverviewViewModelTest.swift

```diff
@@ -120,7 +131,7 @@ class DiaryOverviewViewModelTest: XCTestCase {
 			numberOfDaysWithLowRisk: 1,
 			numberOfDaysWithHighRisk: 1,
 			calculationDate: today,
-			riskLevelPerDate: [todayMinus5Days: .low],
+			riskLevelPerDate: [todayMinus5DaysAfterCheckingDaylightSaving: .low],
```

Comment:
The RiskCalculationResult is based on UTC.
The todayString is calculated based on a UTC formatter.
It would be fine to use a UTC calendar here and not to deal with timezone hiccups.